api/elastic.py

```diff
@@ -60,16 +61,20 @@ def generateSearchQuery(self, params):
                 searchClauses.append(Q('bool', should=[
                     self.titleQuery(escapedQuery),
                     self.authorQuery(escapedQuery),
-                    self.subjectQuery(escapedQuery)
+                    self.subjectQuery(escapedQuery),
+                    self.identifierQuery(escapedQuery),
+                    self.authorityQuery(escapedQuery)
```

Comment:
I think we should leave this out of this general query. The authority query is more specific and is mostly used internally.

Reply:
Ok. I'll delete lines 65 and 66.